### PR TITLE
Make emails images full size and use copyright HTML code

### PR DESCRIPTION
**Pull request by constanzauanini**
Images were not displayed in full size and Copyright symbol was not showing correctly:

Screenshot 2024-06-18 at 6 07 46 PM

### CODE REVIEW

File: src/main/resources/templates/email/user/addedToOrganization/body.ftlh.mjml [hunk 1]
```diff
@@ -23,7 +23,7 @@
                     ${strings("notification.user.addedToOrganization.email.body.2", organization.name)}
                 </mj-text>
                 <mj-text mj-class="text-body03" padding-top="32px">
-                ${strings("notification.user.addedToOrganization.email.body.3", admin.fullName, admin.email)}</mj-text>
+                ${strings("notification.user.addedToOrganization.email.body.3", admin.fullName)}</mj-text>
```
Comment (constanzauanini) [inline src/main/resources/templates/email/user/addedToOrganization/body.ftlh.mjml:26]: @sgrimm I need to show the user name here, not the admin name. Do you know if it is available here?

Comment (sgrimm) [reply src/main/resources/templates/email/user/addedToOrganization/body.ftlh.mjml:26]: Not at the moment, but it's easy to add to the model. I'll do that in a separate PR right now. Up to you whether you want to merge this and then do a followup, or wait for my PR and rebase this one.

**Review by sgrimm — APPROVED** [review]

**Comment by sgrimm** [comment]
#2079 adds a `user` object to the model the template is able to access. To get the user's full name, you'd do `user.fullName`. There are also `firstName` and `lastName` fields if you want one of those instead.

File: src/main/resources/templates/email/user/addedToOrganization/body.ftlh.mjml (hunk 1 above)
Comment (constanzauanini) [reply src/main/resources/templates/email/user/addedToOrganization/body.ftlh.mjml:26]: Thanks, I will merge this one and create a new one after your PR is merged

**Pull request merged**